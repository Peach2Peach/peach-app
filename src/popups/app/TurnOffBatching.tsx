import { PeachText } from '../../components/text/PeachText'
import i18n from '../../utils/i18n'

export const TurnOffBatching = () => <PeachText>{i18n('settings.batching.turnOff.description')}</PeachText>
