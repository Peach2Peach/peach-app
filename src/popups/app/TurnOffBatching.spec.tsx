import { createRenderer } from 'react-test-renderer/shallow'
import { TurnOffBatching } from './TurnOffBatching'

describe('TurnOffBatching', () => {
  const shallowRenderer = createRenderer()
  it('should render correctly', () => {
    shallowRenderer.render(<TurnOffBatching />)
    expect(shallowRenderer.getRenderOutput()).toMatchSnapshot()
  })
})
