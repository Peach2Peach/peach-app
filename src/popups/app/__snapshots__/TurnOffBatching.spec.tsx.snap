// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TurnOffBatching should render correctly 1`] = `
<PeachText>
  You have a pending payout. Turning off batching right now will pay out this escrow right away, at higher fees.

Are youÂ sure?
</PeachText>
`;
